LibVLCSharp/Shared/LibVLC.cs

```diff
@@ -562,8 +569,7 @@ public void UnsetDialogHandlers()
         {
             if (DialogHandlersSet)
             {
-                _dialogCbs = default;
-                Native.LibVLCDialogSetCallbacks(NativeReference, _dialogCbs, IntPtr.Zero);
+                Native.LibVLCDialogSetCallbacks(NativeReference, default, IntPtr.Zero);
```

Comment:
Should we set the struct to `default` as well?

Reply:
passing `default` works as is. The structure doesn't need to change.